pdp-server/src/config/opa.rs

```diff
@@ -0,0 +1,22 @@
+use serde::Deserialize;
+
+/// Configuration for the OPA service
+#[derive(Debug, Deserialize, Clone)]
+pub struct OpaConfig {
+    /// The URL of the OPA service (default: http://localhost:8181)
+    #[serde(default)]
+    pub url: String,
+
+    /// The timeout for OPA client queries in seconds (default: 1)
+    #[serde(default)]
+    pub query_timeout: u64,
+}
+
+impl Default for OpaConfig {
+    fn default() -> Self {
+        Self {
+            url: "http://localhost:8181".to_string(),
+            query_timeout: 1, // 1 second
```

Comment:
this should use also use [PDP_OPA_CLIENT_QUERY_TIMEOUT](https://docs.permit.io/concepts/pdp/configuration/#pdp_opa_client_query_timeout) to be backward compatible